px-vis-timeseries.html

```diff
@@ -1636,8 +1636,14 @@
           this._deleteStripe();
 
           // mock a extentsDataChange to pass data to the stripe creation fn
-          if(!this.extentsData) { this.extentsData = {}; }
-          this.extentsData.eX = [this.x.invert(brushExt[0]).getTime(), this.x.invert(brushExt[1]).getTime()];
+          var exData = [this.x.invert(brushExt[0]).getTime(), this.x.invert(brushExt[1]).getTime()];
+
+          if (!this.extentsData) {
+            this.extentsData = {eX: exData};
+          }
+
+          this.extentsData.eX = exData;
```

Comment:
This one was really subtle changes, and it wasn't obvious why. To make it obvious for others / the future, let's:
1. Add a comment about populating `ex` FIRST so that the event doesn't fire with undefined
2. Let's move the final line into an `else` block unless there is good reason for it to also produce the same change on `ex`